OpenSim/Simulation/Model/Frame.h

```diff
@@ -329,6 +329,10 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
         transform used to position the Geometry is that of this Frame. */
     void attachGeometry(OpenSim::Geometry* geom);
 
+    /** Scale any Geometry attached to this Frame. */
+    void scale(const SimTK::Vec3& scaleFactors);
```

Comment:
See above; renaming would also make this function self-documenting.

Reply:
Good point. Done.